src/renderer/vt/state.cpp

```diff
@@ -526,11 +526,13 @@ void VtEngine::SetTerminalCursorTextPosition(const til::point cursor) noexcept
 // - S_OK if we succeeded, else an appropriate HRESULT for failing to allocate or write.
 HRESULT VtEngine::RequestWin32Input() noexcept
 {
+    // On startup we request the modes we require for optimal functioning
+    // (namely win32 input mode and focus event mode).
+    //
     // It's important that any additional modes set here are also mirrored in
     // the AdaptDispatch::HardReset method, since that needs to re-enable them
     // in the connected terminal after passing through an RIS sequence.
-    RETURN_IF_FAILED(_RequestWin32Input());
-    RETURN_IF_FAILED(_RequestFocusEventMode());
+    RETURN_IF_FAILED(_Write("\033[?9001;1004h"));

```

Comment:
lol, straight up yeet them in. I love it actually.

Reply:
It's the same sequence as in `AdaptDispatch::HardReset`, which I felt like makes it easier to find.